Resources/views/forms/template.html.twig

```diff
@@ -228,6 +229,27 @@
                     </div>
                 </div>
             </div>
+
+            <div class="grid-row m-bottom-40">
+                <div class="grid-col-6">
```

Comment:
Would do it like

Col 3                      | Col3              | Col3                    | Col 3                    
---------------- | ----------- | -------------- | -----------------
ReplyTo                  | Attchment      | Notify Email        | Customer Email    

Or if the space is not enough

Col 6 | Col6
------|------
ReplyTo | Attachment 
Notify Email | Customer Email